studio.code-workspace

```diff
@@ -1,17 +1,17 @@
 {
-	"folders": [
-		{
-			"path": "."
-		}
+  "folders": [
+    {
+      "path": "."
+    }
   ],
   "settings": {
     "typescript.surveys.enabled": false,
     "eslint.alwaysShowStatus": true,
     "eslint.validate": [
       "javascript",
       "javascriptreact",
-      { "language": "typescript", "autoFix": true },
-      { "language": "typescriptreact", "autoFix": true}
+      "typescript",
```

Comment:
is this a personal preference or should it be enforced for the whole team?

Reply:
@acn-sbuad This is actually just a smell in the file, the "autoFix": true is apparently  a default setting so the shortname was enough. (TLDR: this should not change anything)